DOCS/man/lua.rst

```diff
@@ -726,7 +726,7 @@ strictly part of the guaranteed API.
 
 ``utils.getcwd()``
     Returns the directory that mpv was launched from. On error, ``nil, error``
-    is returned.
+    is returned. Retrieving the ``working-directory`` property is preferred.
```

Comment:
If I'm reading this without having any prior knowledge of what these are, the immediate question I'd think of is, "What's the difference between the properties and the util functions and why is properties preferred over it?"

And even as someone who has some knowledge on it, I'm still confused by why properties would be preferred when they are identical.